docs/EN_US/ECLProgrammersGuide/PRG_Mods/PrGit_Source_Code.xml

```diff
@@ -0,0 +1,556 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<sect1 id="Src-code-git">
+  <title>The Git Repository</title>
+
+  <para>Git is free open source software for distributed version control. Git
+  tracks changes for any set of files. With Git every Git directory on every
+  computer is a full-fledged repository with complete history and full
+  version-tracking abilities.</para>
+
+  <para>Refer to <ulink
+  url="https://git-scm.com/">https://git-scm.com/</ulink> for more
+  information.</para>
+
+  <para>HPCC Systems has support for the ECLCC Server to compile ECL code
+  directly from Git repositories. The repositories (and optional
+  branches/users) are configured using environment variables on the server.
+  You can submit a query from a repository branch, and the ECLCC Server will
+  pull the source code from a Git repository and compile it. This allows you
+  to deploy a specific version of a query without needing to perform any work
+  on the client.</para>
+
+  <sect2 id="TheGitImrpovements">
+    <title>The Git Improvements</title>
+
+    <para>Starting with version 8.4, the platform code for Git support
+    significantly improved. Some of these improvements have been backported to
+    older support releases such, as 7.12. However, You still need to update to
+    a recent point release to ensure you get any of these improvements. While
+    the later releases such as 8.6 will include all of these
+    improvements.</para>
+
+    <sect3 id="TheSPeedIMprovements">
```

Comment:
I don't know if it matters, but slightly strange capitalisation in the id.

Reply:
it does not matter